.circleci/config.yml

```diff
@@ -36,8 +42,9 @@ jobs:
       - run: cat packages/*/package.json | shasum > .dependencies_checksum
       - restore_cache:
           key: dependency-cache-{{ checksum ".dependencies_checksum" }}
+      - attach_workspace:
+          at: /home/circleci/project
       - run: yarn # symlink packages' node_modules
```

Comment:
```suggestion
```

Reply:
Don't need it anymore?

Reply:
I don't think so right? Wouldn't `yarn build` fail if the dependencies weren't installed?